### ISSUE TITLE
Rallying Cry of the Dragonslayer missing Internal CD (onyxia/nefarian head)

### ISSUE THREAD

**Issue by knasen88** [elysium-project/server#1529]
> The server or game patch that your issue pertains to (e.g. Anathema, patch 1.10).
Elysium, 1.4 and 1.6 (onyxia head and nef head)

> A clear, concise description of the current behaviour.
Currently, you can turn in the buff back to back even if there is a head up in the chains/on the pole and recieve the buff

> A description of the expected behaviour.
When placed upon the spike/chain, it should not fire any more new buffs until the previous has despawned after X hours, whereas X is uncertain but believed to be around 6 hours (same as WCB 21600(000))
This would mean 2 buff opportunities per X hour (currently only 1 on elysium but Nefarian is soon added)
I suggest copying the WCB script and mimmicking it to work with the heads somehow.

> Where applicable, provide relevant sources that show or describe the correct behaviour.
Patch 1.7 also mentions it briefly that to buff, there can be no (same) head hanging.
https://web.archive.org/web/20070602172347/http://www.worldofwarcraft.com/patchnotes/patch-05-09-13.html

http://alt.games.warcraft.narkive.com/nbkesSJ1/turing-in-onyxia-head-turnin-city-buffs

> When I have handed-in Onyxia's head, nobody was buffed,
because a head already dangled from a stone ridge on the
entrance to Stormwind (valley of heroes). I don't know
how often it is removed, but I've heard that only one head
can be handed-in per week.
Not at all! It's a couple of hours normally (or is that 24h? can't
remember), check the guards to see what buff they have, this will let
you know if your head will buff everyone around.
AFAIK* you can get the buff only every six hours. But the buff (and the
head itself) are only two hours. But I think Nefarians head is on a
different counter; but you can't receive the buff more than once.

**Comment by thepparker** [comment]
Having the 6 hour cooldown doesn't really make sense with a population the size of Elysium's to be honest. With over 50 guilds raiding per faction, that's a large number of Onyxia heads being turned in every day.

Not being able to acquire the buff because one of them was turned in a few hours before your raid (which will happen very often) leaves a bit of a sour taste.

**Comment by fgsfdss** [comment]
Though that same argument applies for Warchief's Blessing, which also has said cooldown and in a bugged state at that (requires two of the quest to be turned in before the CD starts ticking).

**Comment by thepparker** [comment]
I agree, and I don't think the blessing should have such a long cooldown either.

**Comment by strudelkuchen** [comment]
> I agree, and I don't think the blessing should have such a long cooldown either.

^agreed

**Comment by SiguDS** [comment]
You weren't able to get the buff when there was already a head hanging on a spike afaik.

http://wow.allakhazam.com/forum.html?forum=21&mid=114787952210513407
"Dragonslayer is Onyxia's head. 


Horde also get Warchief's Blessing (if you're in Orgrimmar and Crossroads) when someone turns in Rend Blackhand's head. 


Note, each buff is only cast once every 6+ hours (not sure of the time) so you can't get the buff and then have someone turn in another head of Rend (for instance) an hour later. 
"

**Comment by Tunesisback** [comment]
Reposting a comment I made on the #297 issue you linked, SiguDS.

The same post (direct link http://wow.allakhazam.com/forum.html?forum=21&mid=114787952210513407#5) that you quoted says "not sure of the time" when the six hour timeframe is given.  In fairness I will admit that this is a second source that mentions the 6 hour cooldown, but since both the newsgroup archive I mention below and that allakhazam archive both qualify the six hours by saying that the poster isn't sure or didn't confirm, I still think this is weak evidence.

I've let alone the point but i'l bring it up now. Elysium is not strictly or objectively a blizzlike server.  There have already been a great deal of non-blizzlike deviations. Even if the buff can be strongly proved to have such a long cooldown, given the high population, it would be a grave mistake to implement it.

> I signed up for github just for this.  In regards to the source about the cooldown, i'm very skeptical that this is accurate at all.
> 
> from: http://alt.games.warcraft.narkive.com/nbkesSJ1/turing-in-onyxia-head-turnin-city-buffs#post9
> 
> > AFAIK* you can get the buff only every six hours. But the buff (and the
> > head itself) are only two hours. But I think Nefarians head is on a
> > different counter; but you can't receive the buff more than once.
> > 
> > * I haven't stood around for 6 hours and tried to hand in the head etc.
> > 
> > Urs
> 
> This guy is speculating that it's 6 hours, and makes it clear that he hasn't tested the 6h time frame.  This poster, Urs, also states that you can't recieve the buff more than once, which I'm certain is false.  Since he's stated one false item and has 1 speculative item I don't think he knew what he was talking about.
> 
> Nothing I see in the twinstar links suggest a 6h internal cooldown, or otherwise suggests a duration of the onyxia/nefarian head monument in org/sw.
> 
> There are indications in that newsgroup archive that the buff can't be received while the monument is spawned (http://alt.games.warcraft.narkive.com/nbkesSJ1/turing-in-onyxia-head-turnin-city-buffs#post3), I really think that the newsgroup archive comments about a 6h despawn is a very weak source.
> 
> I'm going to repost this comment on the other relevant issue.

**Comment by knasen88** [comment]
Then it would only be fair that warchief's CD was removed aswell. Not only is it harder to achieve but with current limitations so many WCB gets wasted

**Comment by Tunesisback** [comment]
> Exact the same argument was made for Warchiefs Blessing,
A complete random post from a random poster on a datamined forum from 10 years ago was the "argument" for warchief's blessing and somehow, for some fucking reason, was enough "evidence" to get a CD on Elysium PVP.
Let's do the same for ony buff for both factions, or just remove the stupid CD for warchief's blessing and keep Rallying cry of the dragonslayer as it is, atleast until' naxx is on the table for each server.

@jessemon i don't doubt it, but can you link to the issue? can't find it.  I'd like to see the comments.  I see #343 and #179 but i don't see any comments there like you said

**Comment by knasen88** [comment]
@Tunesisback alleged source of WCB getting 6h CD 

> The event/buff should be on a 6 hours cooldown.
http://wow.allakhazam.com/forum.html?forum=21&mid=113690196285182144

**Comment by SiguDS** [comment]
There were arguments made (from Elysium) the population shouldn't matter since there isn't any cap for how MANY can get the buff. 

But the amount of different raiding hours (china in the mid day, EU afternoon/start of the night and NA at night it would cause commotion probably. I don't think the ony buffs would be as bad because you could easily see if the head is up and you have BOTH ony and nef heads that could be used. Compared to rend head it's also easy to see if it will give a buff (since rend head doesn't have a head on a spike). 

Random Sigu rant:
Hard decision to make for elysium though because on one hand a lot of people don't like all this world buff stuff but only giving it 6 hour cooldown is just making it harder to plan for but not impossible. I like the idea of no world buffs at all when you enter a raid but that is obviously too far away from "blizzlike" vanilla. The backlash from putting cooldown on them i feel would be fairly high since a lot of guilds use the buff to give smoother raids. I feel there would have been a crazy backlash for rend buff having cd if we have had rend buff since the start back at nost. but since it got added with the cooldown people weren't as annoyed by it, (or atleast didn't speak up about it). A lot of the issues with adding cooldown to ony/nef are the issues that are ALREADY on rend head (except the lack of visual for when rend head works).

**Comment by Tunesisback** [comment]
@knasen88 OK, I read through the source and these are the relevent mentions.  I don't like any of the langauge used, none of it appears to me as authorative.

http://wow.allakhazam.com/forum.html?forum=21&mid=113690196285182144#3

> I'm not positive but I think the buff can't go off every hour. I think it can only be given by Thrall once every 6 hours. 

"I think" isn't evidence of anything.

http://wow.allakhazam.com/forum.html?forum=21&mid=113690196285182144#10

> edit: It's too hard to time when you get the buff. It's either once a day or every 6 hours (the 6 hour one is correct I'm thinking now..). 

This commenter only says 6 hours because someone earlier in the thread said so.  This commenter wasn't sure if it was every 24h or every 6h, a HUGE difference that would easily be confirmable if the player had any idea what they were talking about.

Further down in the same post (number 10)

> Can you really get thru to Rag or Nef in 2 hours after being in Org with your whole raid? Some guilds could, but doubtful... 

This raises some questions.  MC in over 2h? I wouldn't include this poster in the top tier of player from that era, so I begin to doubt whether or not they would be acutely aware of the buff timers.

http://wow.allakhazam.com/forum.html?forum=21&mid=113690196285182144&h=50&p=1#16

> Yeah..6 hours seems accurate. On azshara we get it about twice a day every day.

Again, it's just the same kind of language.  "seems, about" are qualifying words.  I don't doubt there was a cooldown, i just have't seen anything in these sources that convinces me it's six hours.

However, again, this ends up beside my point earlier.

> Elysium is not strictly or objectively a blizzlike server. There have already been a great deal of non-blizzlike deviations. Even if the buff can be strongly proved to have such a long cooldown, given the high population, it would be a grave mistake to implement it.

@SiguDS hey, you said something in your rant I thought was odd, where you mention that a lot of people "don't like all this world buff stuff".  which people are you talking about? what does a lot mean? Every serious raiding guild I've heard of, not just on Elysium, but on Kronos, on Nostalrius, on Rebirth, getting world buffs is a requirement.  The exceptions to this are edge cases where the guild is very casual and don't raid at a high level (i.e. speed clears/speed kills)

**Comment by SiguDS** [comment]
Well @Tunesisback i guess it might have just been the people i have been around. But since every guild gets the buffs then wouldn't it be the same competition if no one has them? I mean i get that it puts a difference between casual guilds and hardcore guilds but between hardcore guilds and other hardcore guilds the competition would be the same just less time spent on that shit :) anyway i get what you mean though.
